### PR TITLE
Update MGW related configs for operator 1.2.0 release

### PR DESCRIPTION
**Pull request by RakhithaRR**
### Purpose
This PR, 

- Adds eventhub configs for MGW to validate subscriptions.

- Updates jms connection URL for global throttling.

- Updates the default token to a token obtained from APIM 3.2.0

### CODE REVIEW

File: api-operator/pkg/mgw/config.go [hunk 1]
```diff
@@ -64,6 +64,7 @@ const (
 	enableRealtimeMessageRetrievalConst = "enableRealtimeMessageRetrieval"
 	enableRequestValidationConst        = "enableRequestValidation"
 	enableResponseValidationConst       = "enableResponseValidation"
+	enabledEventhub						= "enabledEventhub"
```
Comment (menakaj) [inline api-operator/pkg/mgw/config.go:67]: Formatting issue

File: api-operator/deploy/controller-configs/controller_conf.yaml [hunk 2]
```diff
@@ -21,9 +21,9 @@ metadata:
   namespace: wso2-system
 data:
   #mgw toolkit image to initialize/setup the micro gw project
-  mgwToolkitImg: wso2am/wso2micro-gw-toolkit:3.2.0-beta
+  mgwToolkitImg: rakhitharr/wso2micro-gw-toolkit:3.2.0-custom1
   #mgw runtime image to use in the mgw executable
-  mgwRuntimeImg: wso2/wso2micro-gw:3.2.0-beta
+  mgwRuntimeImg: rakhitharr/wso2micro-gw:3.2.0-custom1
```
Comment (menakaj) [inline api-operator/deploy/controller-configs/controller_conf.yaml:26]: Shall we remove the user-specific location?

File: api-operator/pkg/mgw/config.go (hunk 1 above)
Comment (RakhithaRR) [reply api-operator/pkg/mgw/config.go:67]: Fixed in latest code

File: api-operator/deploy/controller-configs/controller_conf.yaml (hunk 2 above)
Comment (RakhithaRR) [reply api-operator/deploy/controller-configs/controller_conf.yaml:26]: Those were added only for pre release testing purposes. Removed in the latest code.

**Pull request merged**